website/static/js/pages/reg-landing-page.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
```

Comment:
A lot of this seems duplicated from the prereg page. Can you put the shared JS in a re-usable JS module that is imported on both pages?